jvm.ini in jetty/base/start.d is throwing an error related to "xalan" in case of upgrading the CMS version

**Describe the bug**

jvm.ini in "jetty/base/start.d" is throwing an error related to "Provider org.apache.xalan.xsltc.trax.SmartTransformerFactoryImpl not found" in case of upgrading the CMS version.

**To Reproduce**

Steps to reproduce the behavior:
1. Install 8.0.3 version of CMS.
2. Then upgrade to 8.1.2 version and start jetty.
3. Jetty will not start and it will throw an error.
4. I am enclosing "server.log" file to understand the error better. 
[server.log](https://github.com/percussion/percussioncms/files/13229152/server.log)


**Expected behavior**

Jetty should start and error should not come.

**Work Around**

Open jvm.ini file in (jetty/base/start.d) and remove lines(36 to 39):
**-Djavax.xml.transform.TransformerFactory=org.apache.xalan.xsltc.trax.SmartTransformerFactoryImpl
-Djavax.xml.parsers.SAXParserFactory=com.sun.org.apache.xerces.internal.jaxp.SAXParserFactoryImpl
-Djavax.xml.datatype.DatatypeFactory=com.sun.org.apache.xerces.internal.jaxp.datatype.DatatypeFactoryImpl
-Djavax.xml.parsers.DocumentBuilderFactory=com.percussion.xml.PSDocumentBuilderFactoryImpl**

**Fix**

Go to "modules\perc-distribution-tree\src\main\resources\distribution\rxconfig\Installer\install.xml" in your project and delete the code from line 1265 to 1269
**copy todir="${install.dir}/jetty/base/start.d" failonerror="false" overwrite="false"
fileset dir="${install.src}/jetty/defaults/start.d" followsymlinks="false" includes="jvm.ini"
present present="srconly" targetdir="${install.dir}/jetty/base/start.d"
/fileset
/copy**

And instead of that we have to write the code to create empty jvm.ini file in "jetty/base/start.d" if it is not there.

Failed during testing:
Caused by: java.lang.ClassNotFoundException: org/apache/xalan/xsltc/trax/SmartTransformerFactoryImpl
        at java.lang.Class.forName0(Native Method) ~[?:1.8.0_371]
        at java.lang.Class.forName(Unknown Source) ~[?:1.8.0_371]
        at javax.xml.transform.FactoryFinder.getProviderClass(Unknown Source) ~[?:1.8.0_371]
        at javax.xml.transform.FactoryFinder.newInstance(Unknown Source) ~[?:1.8.0_371]
        at javax.xml.transform.FactoryFinder.find(Unknown Source) ~[?:1.8.0_371]
        at javax.xml.transform.TransformerFactory.newInstance(Unknown Source) ~[?:1.8.0_371]
        at org.springframework.http.converter.xml.SourceHttpMessageConverter.<init>(SourceHttpMessageConverter.java:87) ~[?:?]
        at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:1.8.0_371]
        at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source) ~[?:1.8.0_371]
        at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source) ~[?:1.8.0_371]
        at java.lang.reflect.Constructor.newInstance(Unknown Source) ~[?:1.8.0_371]
        at org.springframework.beans.BeanUtils.instantiateClass(BeanUtils.java:211) ~[?:?]
        at org.springframework.beans.factory.support.SimpleInstantiationStrategy.instantiate(SimpleInstantiationStrategy.java:87) ~[?:?]
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.instantiateBean(AbstractAutowireCapableBeanFactory.java:1326) ~[?:?]
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1232) ~[?:?]
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:582) ~[?:?]
        at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:542) ~[?:?]
        at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveInnerBean(BeanDefinitionValueResolver.java:374) ~[?:?]
        ... 60 more

--- a/modules/perc-distribution-tree/src/main/resources/distribution/rxconfig/Installer/updateConfiguration.xml
+++ b/modules/perc-distribution-tree/src/main/resources/distribution/rxconfig/Installer/updateConfiguration.xml
@@ -232,6 +232,10 @@
             <available file="${install.dir}/jetty/base/start.d/jvm.ini"/>
             <then>
                 <replace file="${install.dir}/jetty/base/start.d/jvm.ini" token="@@rxdir@@" value="${normalized.install.dir}" />
+                <replace file="${install.dir}/jetty/base/start.d/jvm.ini" token="-Djavax.xml.transform.TransformerFactory" value="#-Djavax.xml.transform.TransformerFactory" />
+                <replace file="${install.dir}/jetty/base/start.d/jvm.ini" token="-Djavax.xml.parsers.SAXParserFactory" value="#-Djavax.xml.parsers.SAXParserFactory" />
+                <replace file="${install.dir}/jetty/base/start.d/jvm.ini" token="-Djavax.xml.datatype.DatatypeFactory" value="#-Djavax.xml.datatype.DatatypeFactory" />
+                <replace file="${install.dir}/jetty/base/start.d/jvm.ini" token="-Djavax.xml.parsers.DocumentBuilderFactory" value="#-Djavax.xml.parsers.DocumentBuilderFactory" />
             </then>
         </if>
         <replace file="${install.dir}/PercussionXMLCatalog.xml" token="@@rxdir@@" value="${lead.slash}${normalized.install.dir}" />

